frontend/src/component/project/Project/ProjectFeatureToggles/FeatureToggleSwitch/useFeatureToggleSwitch.tsx

```diff
@@ -96,7 +96,11 @@ export const useFeatureToggleSwitch: UseFeatureToggleSwitchType = (
             };
 
             const ensureActiveStrategies: Middleware = (next) => {
-                if (!config.hasStrategies || config.hasEnabledStrategies) {
+                if (
+                    newState === false ||
```

Comment:
Other conditions should never be met with `newState===false`, but adding this makes it more consistent with our actual business logic